docs/destinations/forwarding-events/snowbridge/configuration/transformations/custom-scripts/create-a-script/index.md

```diff
@@ -86,22 +74,22 @@ To set the Partition Key in the message, you can simply set the input's Partitio
 https://github.com/snowplow/snowbridge/blob/master/assets/docs/configuration/transformations/custom-scripts/create-a-script-setpk-example.js
 ```
 
-```lua reference
-https://github.com/snowplow/snowbridge/blob/master/assets/docs/configuration/transformations/custom-scripts/create-a-script-modify-example.lua
-```
-
 Or, if modifying the data as well, return the modified data and PartitionKey field:
 
 ```js reference
 https://github.com/snowplow/snowbridge/blob/master/assets/docs/configuration/transformations/custom-scripts/create-a-script-setpk-modify-example.js
 ```
 
-```lua reference
-https://github.com/snowplow/snowbridge/blob/master/assets/docs/configuration/transformations/custom-scripts/create-a-script-setpk-modify-example.lua
+## Setting a HTTP header (v)
```

Comment:
what’s the (v)?

Reply:
Typo. And actually your other comment is where it came from.